### PR TITLE
Navbar focustrap

### PR DESCRIPTION
**Pull request by viggo**

### CODE REVIEW

File: packages/react/src/navbar/navbar-expandable-menu.tsx [hunk 1]
```diff
@@ -25,34 +26,21 @@ export function NavbarExpandableMenu({ children }: NavbarExpandableMenuProps) {
...
-NavbarExpandableMenu.displayName = "NavbarExpandableMenu";
-/*
-function hideAllOtherElements() {
-  //const elements = document.querySelectorAll("body > *");
   const navbarElement = document.getElementsByClassName(clsx("hds-navbar"))[0];
-  const siblings = getAllSiblings(navbarElement);
-  console.log(siblings);
+  return (
+    <navbarContext.Provider value={[open, toggleOpen]}>
+      {open ? <FocusTrap containerElements={[navbarElement as HTMLElement]} /> : false}
```
Comment (haraldvinje) [inline packages/react/src/navbar/navbar-expandable-menu.tsx:37]:
```suggestion
      {open ? <FocusTrap containerElements={[navbarElement as HTMLElement]} /> : null}
```

**Review by haraldvinje — DISMISSED** [review]
Small comment, otherwise LGTM 🚀

File: packages/react/src/navbar/navbar-expandable-menu.tsx (hunk 1 above)
Comment (juliuxu) [reply packages/react/src/navbar/navbar-expandable-menu.tsx:37]: not a big deal, but agree

File: packages/react/package.json [hunk 2]
```diff
@@ -23,6 +23,7 @@
     "@types/react": "^18.2.67",
     "@types/react-dom": "^18.2.22",
     "eslint-config-custom": "workspace:*",
+    "focus-trap-react": "10.2.3",
```
Comment (juliuxu) [inline packages/react/package.json:26]: This needs to be in the `dependencies`, not `devDependencies`. Updating...

**Review by juliuxu — APPROVED** [review]
🚀

**Review by haraldvinje — APPROVED** [review]

**Pull request merged**